resources/views/components/site-stats-widget/server-metrics-item.blade.php

```diff
@@ -1,4 +1,6 @@
 @php
+    $siteMetrics = $site->serverMetrics;
+    
     $cpuLoad = $site->serverMetrics->first()->cpu_load ?? null;
     $ramUsage = $site->serverMetrics->first()->memory_usage ?? null;
     $diskUsage = $site->serverMetrics->first()->disk_usage['percentage'] ?? null;
```

Comment:
You added a new variable named `$siteMetrics` but you're not even using it.

```suggestion
    $cpuLoad = $site->latestServerMetric->cpu_load ?? null;
    $ramUsage = $site->latestServerMetric->memory_usage ?? null;
    $diskUsage = $site->latestServerMetric->disk_usage['percentage'] ?? null;
```